invisible_cities/config/penthesilea.conf

```diff
@@ -10,7 +10,7 @@ include('$ICDIR/config/s12_selector.conf')
 
 # override the default input/output files:
 
-files_in = '$ICDIR/database/test_data/dst_NEXT_v1_00_05_Kr_ACTIVE_0_0_7bar_PMP_10evt.h5'
+files_in = '$ICDIR/database/test_data/Kr_7bar_pmaps.h5'
```

Comment:
Yes, I produced some new files specifically, since we haven't done an official production yet. The files are ususlly small, I can upload them here, too. Should I upload also the corresponding MCRD and the RWF files?

Reply:
In the end, since we have by now a first tag of nexus with hdf5 output, I decided to generate again those files and call them with a meaningful naming. I have also uploaded the original nexus files and the MCRD and RWF ones.